libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -191,6 +199,8 @@ singleton RangeFinder method status_orient uint8_t Rotation'enum ROTATION_NONE R
 singleton RangeFinder method has_data_orient boolean Rotation'enum ROTATION_NONE ROTATION_MAX-1
 singleton RangeFinder method get_pos_offset_orient Vector3f Rotation'enum ROTATION_NONE ROTATION_MAX-1
 
+singleton RangeFinder method get_backend AP_RangeFinder_Backend uint8_t 0 RANGEFINDER_MAX_INSTANCES-1
```

Comment:
should be skip_check, and change the other rangefinder ones to skip_check, and use nil check in script